rfcs/0167-Easy-Taskcluster-setup.md

```diff
@@ -0,0 +1,44 @@
+
+## Assumptions
+
+The solution can be repository host agnostic, but the details of this RFC are based on the Taskcluster integration with Github.
+
+
+# Details
+
+Create a tool or feature that will automatically generate a yaml config file as part of the taskcluster-github app installation and setup. This tool would have presets for common use cases such as running javascript tests or python tests. This might look like a series of questions in a form to help users get started with some default scope, worker type configurations and basic commands.
+
+A second part of this task would be to improve the Github Integration section of the Taskcluster docs to provide details around the installation of the community and firefox-ci apps with github, and clearer guidance around scopes and worker types (for advanced use cases).
+
+## Implementation
+
+How this tool or feature will be implemented is an open question. There are a few different approaches that have been suggested (other idea are also welcome):
+1. After the user installs the taskcluster github app, users will be redirected to a new taskcluster view or other webapp that will walk the user through config setup and commit the config to the repository. 
+
+The advantage of this approach is that it would be in line with what a least one of our competitors offers and would open the door to creating additional value for our users (including where Push Health might fit in). To keep this in scope, additional features - such as what users expect to see or would like to have - can be explored during initial user research and broken down into future milestones.
+
+2. After installing the taskcluster-github app, the user is redirected back to the taskcluster docs. A feature integrated in the UI would walk users through config file setup and generate a yaml file that could be copied into their codebase.
+
+3. The user installs a CLI tool and answers a series of prompts on the command line that then adds the config file to the root directory.
+
+## Strategy
+
+If there is uncertainty around which approach to take, we could conduct an informal survey of the different approaches to a small group of our target users to gauge general expectations and preferences.
+
+Once we decide on an approach, a suggested strategy to ensure a strong user experience would be to create a basic code or figma mockup and solicit additional feedback from our test group of users. This could be presented to each user in such a way so that the user is observed interacting with the mockup/code (or CLI tool) over a video call and any feedback and observations can be used to guide subsequent iterations (basically, until the feature has reached a point where a majority of users find it intuitive to use).
```

Comment:
I think this is an awesome idea. The success of this heavily depends on the UX IMO, so this sort of step has a lot of value.